MovieApp/MovieApp/App/Features/MovieBrowser/NetworkClient/MovieClient.swift

```diff
@@ -35,10 +36,23 @@ class MovieClient: MovieClientProtocol {
         }
     }
     
-    func fetchMovieDetails(for movieId: Int, completion: @escaping(Result<MovieDetailResponse, RequestError>) -> Void) {
-        fetch(forUrl: "movie/\(movieId)") { (result: Result<MovieDetailResponse, RequestError>) in
-            completion(result)
+//    func fetchMovieDetails(for movieId: Int, completion: @escaping(Result<MovieDetailResponse, RequestError>) -> Void) {
+//        fetch(forUrl: "movie/\(movieId)") { (result: Result<MovieDetailResponse, RequestError>) in
+//            completion(result)
+//        }
+//    }
```

Comment:
no need for this anymore

Reply:
[Deleted](https://github.com/lihter/Movie-App/pull/47/files#diff-0c98b99535b0a58a1773d03d140829a86847842a1642eeaf88fade3615fa98f5L39-L43)